cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -79,7 +79,10 @@ def _is_single(self) -> bool | None:
         """
         return (
             self.adata.uns["spatial"]["is_single"]
-            if hasattr(self.adata, "uns") and "spatial" in self.adata.uns and "is_single" in self.adata.uns["spatial"]
+            if hasattr(self.adata, "uns")
+            and "spatial" in self.adata.uns
+            and self.adata.uns["spatial"]
+            and "is_single" in self.adata.uns["spatial"]
```

Comment:
@Bento007 `None` now passes 
But setting `uns['spatial']` = to `1`, `1.0`, or `True` raises this TypeError at this line: `TypeError: argument of type '{ int | float | bool}' is not iterable`